web3/manager.py

```diff
@@ -139,13 +137,11 @@ def default_middlewares(w3: "Web3") -> List[Tuple[Middleware, str]]:
         Leaving ens unspecified will prevent the middleware from resolving names.
         """
         return [
-            (request_parameter_normalizer, "request_param_normalizer"),  # Delete
             (gas_price_strategy_middleware, "gas_price_strategy"),
             (name_to_address_middleware(w3), "name_to_address"),  # Add Async
             (attrdict_middleware, "attrdict"),
-            (pythonic_middleware, "pythonic"),  # Delete
             (validation_middleware, "validation"),
-            (abi_middleware, "abi"),  # Delete
+            (abi_middleware, "abi"),
```

Comment:
Couldnt delete this just yet, removal causes validation errors due to invalid formats (string not converted to hex, etc). 
Tests failing in this build without `abi_middleware`: https://app.circleci.com/pipelines/github/ethereum/web3.py/4559/workflows/8c5fe87f-fe5f-4a45-97ee-8391158c502f/jobs/308436

Reply:
This can be a separate PR, but since all of the tests are failing on the `eth-tester` inbound validation, I suspect there is a bug with either the inbound validation in `eth-tester`, or with our `eth-tester` logic that does some type conversions before we send to `eth-tester`. We should make an issue though to track this removal.